cumulusci/tasks/push/tests/test_push_tasks.py

```diff
@@ -0,0 +1,481 @@
+            "version": VERSION,
+            "start_time": "2020-06-10T10:15",
+            "namespace": NAMESPACE,
+        },
+    )
+    task.push = mock.MagicMock()
+    with pytest.raises(CumulusCIException):
+        task._run_task()
+    os.remove(ORG_FILE)
+
+
+def test_load_orgs_file():
+    # creating sample org file for testing
+    # testing with empty file
+    with open(ORG_FILE, "w") as file:
+        file.write(f"{ORG_FILE_TEXT}")
+        task = create_task(BaseSalesforcePushTask, options={})
+        assert task._load_orgs_file(ORG_FILE) == []
+
+    # testing with multiple orgs
+    with open(ORG_FILE, "r") as file:
+        assert task._load_orgs_file(ORG_FILE) == [
+            "00DS0000003TJJ6MAO",
+            "00DS0000003TJJ6MAL",
+        ]
+    os.remove(ORG_FILE)
+
+
+def test_get_subs_raises_err():
+    task = create_task(GetSubscriberList, options={"filename": "in.csv"})
+    with pytest.raises(NotImplementedError):
+        task()
+
+
+def test_filter_subs_raises_err():
+    task = create_task(
+        FilterSubscriberList, options={"file_in": "in.csv", "file_out": "out.txt"}
+    )
+    with pytest.raises(NotImplementedError):
+        task()
+
+
+def test_base_push_task_raises_err():
+    task = create_task(BaseSalesforcePushTask, options={})
+    with pytest.raises(NotImplementedError):
+        task()
+
+
+def test_report_push_status_error():
+    task = create_task(BaseSalesforcePushTask, options={})
+    task.sf = mock.MagicMock()
+    task.push_report = mock.MagicMock()
+    task.sf.query_all.return_value = {"totalSize": 1, "records": []}
+    with pytest.raises(PushApiObjectNotFound):
+        task._report_push_status("0DV1R000000k9dEWAQ")
+
+
+def test_get_push_request_job_results(
+    package_push_job_success, package_push_job_failure, package_push_job_cancel
+):
+    task = create_task(BaseSalesforcePushTask, options={})
+    task.sf = mock.MagicMock()
+    task.push_report = mock.MagicMock()
+    task.push_request = mock.MagicMock()
+    task.push_request.get_push_job_objs.return_value = [
+        package_push_job_success,
+        package_push_job_failure,
+        package_push_job_cancel,
+    ]
+    task._get_push_request_job_results()
+    task.push_request.get_push_job_objs.assert_called_once()
+
+
+def test_schedule_push_org_query_get_org_error():
+    task = create_task(
+        SchedulePushOrgQuery,
+        options={
+            "orgs": ORG,
+            "version": VERSION,
+            "namespace": NAMESPACE,
+            "start_time": None,
+            "batch_size": "200",
+            "min_version": "1.1",
+        },
+    )
+    task.push = mock.MagicMock()
+    task.push_api = mock.MagicMock()
+    task.sf = mock.Mock()
+    task.push_api.return_query_records = mock.MagicMock()
+    task.push_api.return_query_records.return_value = {"totalSize": "1"}
+    task.sf.query_all.return_value = PACKAGE_OBJS
+    with pytest.raises(ValueError):
+        assert task._get_orgs() == [NAME]
```

Comment:
What behavior is this trying to test for? I would tend to think a ValueError is a bug since we generally want to show helpful error messages that explain what happened.

Reply:
@jofsky can you show me where the ValueError is coming from?

Reply:
I believe https://github.com/SFDO-Tooling/CumulusCI/blob/7b2d16741eac2bb9b98f9bdb6404f524703710ab/cumulusci/tasks/push/tasks.py#L363